docs/header-files.md

```diff
@@ -26,7 +24,8 @@ from header import omp_get_num_threads
 print('number of threads is :', omp_get_num_threads())
 ```
 Pyccel can compile the Python file with the following command: `pyccel example.py --openmp`
-, It will then create the executable file `example`
+, it will then create the executable file `example`.
```

Comment:
Could we not place the comma at the end of the previous line?

Reply:
If you can't see why this line break is necessary either then I'm removing it: e70a9f958

I assumed there was some kind of problem with a code snippet in the middle of a row. But I use these a lot and I am not aware of any problem